Realm/RLMApp_Private.h

```diff
@@ -31,6 +31,8 @@ typedef void(^RLMAppNotificationBlock)(RLMApp *);
 @end
 
 @interface RLMApp ()
+/// A custom base URL to request against. If not set or set to nil, the default base url for app services will be returned.
+@property (nonatomic, readonly) NSString *baseURL;
```

Comment:
Should this have a `nonnull` annotation? I don't suppose we can ever return `nil` here.

Reply:
When we add the header RLM_HEADER_AUDIT_BEGIN(nullability, sendability) which assumes that all pointers in the region are assumed to be '_Nonnull' by default, you have to specify otherwise